### ISSUE TITLE
Add profiles and modpacks to the official launcher

### ISSUE THREAD

**Issue by Lancelotbronner** [gorilla-devs/ferium#212]
## The problem

It is possible to add the modpack and profiles to the official Minecraft launcher to make launching those profiles easier.

I'd like if ferium could do it automatically!

## My solutions

Maybe a configuration on a profile to automatically sync over to a launcher profile. It would create the launcher profile when creating the profile/modpack and store its id. Whenever it is modified the launcher profile id can be updated!

1. Install the mod loader version to the launcher (using the official installers?)
2. Infer launcher profile name & installation folder from the profile or modpack directory
3. Make the launcher profile version the one just installed
4. (Bonus) Embed the icon into the launcher profile icon!

**Comment by theRookieCoder** [comment]
Since ferium is going to become a CLI Minecraft launcher this doesn't make sense, but I do like the convenience of a feature like this so I'll have to think about it.

**Comment by Lancelotbronner** [comment]
Yeah, it's very much a nice-to-have! I think it would mostly be nice for non-tech savvy users (aka my friends XD) so the CLI usage can be minimized to installation or updates.
